### DISCUSSION TITLE
References: Zotero integration

### DISCUSSION THREAD

**Discussion by tosteiner** [knowledgefutures/pubpub#1568] [Ideas]
@gabestein Hi Gabe, I just remembered we previously had a [conversation about this floating around somewhere](https://copim.pubpub.org/pub/using-pubpub-for-scholarly-writing-citations-collaboration-zotero/), and as I'm convinced a Zotero integration would be an amazing feature to have, I just thought I'd throw this in here as an idea :)

**Comment by gabestein** [comment]
Thanks! Yup, I agree. It's on our radar! We actually just had a discussion about it in the last dev office hours call. We're a little torn between putting effort into building out more of PubPub's citation graph (e.g. a community db of citations) or integrating with other services, but maybe leaning towards the latter, at least in the short term? Long-term, we'd like to do both.

**Reply by tosteiner** [reply]
Thanks, Gabe, that's awesome - would love to see how this pans out!
 
This just crossed my mind, maybe the BibTeX connection might be an interesting link to follow up on in that respect (as PubPub already works with BibTeX). From Zotero's end, there's e.g. the excellent [**Better BibTeX for Zotero**](https://retorque.re/zotero-better-bibtex/) plugin that facilitates improved BibTeX handling in Zotero, and, among other features, has a variety of ["Cite-as-you-write" integrations](https://retorque.re/zotero-better-bibtex/citing/cayw/)
